packages/typescript-estree/tests/ast-alignment/fixtures-to-test.ts

```diff
@@ -371,6 +371,16 @@ tester.addFixturePatternConfig('typescript/basics', {
     'class-with-two-methods-computed-constructor',
     'export-type-star-from',
     'import-type-error',
+
+    /**
+     * Babel's 'typescript' transform gives these TypeScript-like properties
+     * such as `isExport: false`, but we don't include those in our AST.
+     */
+    'import-equal-declaration',
+    'import-export-equal-declaration',
+    'import-equal-type-declaration',
+    'import-export-equal-type-declaration',
```

Comment:
@bradzacher I'm not 100% that this is accurate, but hey - it got the tests to pass! 🙃

Reply:
this is one way to do it.
the other way would be do add a transformer to mutate the AST to match babels.

either way sucks.

this is why the ast-spec explicitly snapshots the differences so we don't need to maintain any hacks